content/en/docs/collector/resiliency.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Resiliency
+description: How to configure a resilient OTel Collector pipeline
+---
+
+The OpenTelemetry Collector is designed with components and configurations to
+minimize data loss during telemetry processing and exporting. However,
+understanding the potential scenarios where data loss can occur and how to
+mitigate them is crucial for a resilient observability pipeline.
+
+## Understanding Collector Resilience
+
+The Collector's resilience primarily revolves around how it handles data when
+the configured endpoint (the destination for traces, metrics, or logs) becomes
+unavailable or when the Collector instance itself experiences issues like
+crashes.
+
+## Internal Queues (In-Memory Buffering)
+
+The most basic form of resilience built into the Collector's exporters is the
+**sending queue**.
+
+- **How it works:** When an exporter is configured, it usually includes a
+  sending queue that buffers data in memory before sending it to the downstream
+  endpoint. If the endpoint is available, data passes through quickly.
+- **Handling Endpoint Unavailability:** If the endpoint becomes unavailable
+  (e.g., network issues, backend restart), the exporter cannot send data
+  immediately. Instead of dropping the data, it adds it to the in-memory sending
+  queue.
```

Comment:
```suggestion
- **Handling Endpoint Unavailability:** If the endpoint becomes unavailable,
  for example due to network issues or a backend restart, the exporter cannot
  send data   immediately. Instead of dropping the data, it adds it to the
  in-memory sending queue.
```